### ISSUE TITLE
New Binding trial errors

### ISSUE THREAD

**Issue by vinfur** [mozilla/geckodriver#850]
In order to help us efficiently investigate your issue, please provide the following information:

## Platform and application details
* Platform: 
* Firefox: 
* Selenium: 

Firstly i would say m creating binding myself because language m using is not supported

Error 1 
Element Selector Strategies
Name And Id Are Not working anymore (available here)(https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#sessionsessionidelement)
Although Chrome and many other drivers allow use of this strategies

Error 2
while getting response from creating new session i get Json response as Value->sessionId

While on Other drivers(chrome) i get it as Sessionid directly

Error 3
While Getting element id on Otherdriver (chrome driver) i get it as value->element
While on Gecko Driver i get it as Value->element(followed here by some random digits)

M using win 7 +Geckodriver marionette

So Basically I Wanted to know is geckodriver doing something wrong or chrome driver or i have to add different bindings for all drivers seperately

**Comment by andreastt** [comment]
geckodriver does not implement the Selenium JSON wire protocol, but the [W3C WebDriver protocol](https://w3c.github.io/webdriver/webdriver-spec.html).

You should note that Selenium provides a shim to talk to drivers speaking either protocol, and a reverse proxy in the Selenium server to convert from one to the other.
